index.js

```diff
@@ -1,3 +1,79 @@
-var binding = require('node-gyp-build')(__dirname);
-module.exports = binding;
+class ShortcutHelper {
+	constructor() {
+			this.impl = require('node-gyp-build')(__dirname);
+			if (process.platform === 'darwin') {
+				this.macImpl = require('node-gyp-build')(__dirname);
+			}
+	}
+
+	start(enableLogger) {
+		return this.impl.start(enableLogger);
+	}
+
+	stop() {
+		return this.impl.stop();
+	}
+
+	reload() {
+		this.impl.unregisterAllShortcuts();
+		this.impl.stop();
+		this.impl.start(false);
+	}
+
+	registerShortcut(keys, callback, releaseCallback, keysAreVKC) {
+		return this.impl.registerShortcut(keys, callback, releaseCallback, keysAreVKC);
+	}
+
+	unregisterShortcut(shortcutId) {
+		return this.impl.unregisterShortcut(shortcutId);
+	}
+
+	collectPressedKeyCodes() {
+		console.log('\r\n(DHK) looking for pressed keys');
+		if (process.platform === 'win32') {
+			throw new TypeError('win32 impl does not track the keys');
+		}
+		this.impl.collectPressedKeyCodes(true);
+		return true;
```

Comment:
Since this always returns true (or throws), we can probably just remove this return value